Sources/Asset.swift

```diff
@@ -12,6 +12,40 @@ import ObjectMapper
 /// An asset represents a media file in Contentful
 public class Asset: Resource, LocalizedResource {
 
+    /// URL of the media file associated with this asset. Optional for compatibility with `select` operator queries.
+    public var urlString: String?
```

Comment:
this is CDA only so far. But, i suppose in the future...yes? I'm only implementing what is listed [here](https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/assets)